CMakeLists.txt

```diff
@@ -357,7 +364,7 @@ else()
 
     if(QUIC_ENABLE_SANITIZERS)
         message(STATUS "Configuring sanitizers")
-        list(APPEND QUIC_COMMON_FLAGS -fsanitize=address,leak,undefined -fsanitize-address-use-after-scope -Og -ggdb3 -fno-omit-frame-pointer -fno-optimize-sibling-calls)
+        list(APPEND QUIC_COMMON_FLAGS -fsanitize=address,leak,undefined -fsanitize-address-use-after-scope -O0 -ggdb3 -fno-omit-frame-pointer -fno-optimize-sibling-calls)
```

Comment:
Why this change?

Reply:
I'm not sure, Max did it.

Reply:
I changed it to -O0 because -Og is almost *never* what you want. -Og is "optimize for debuggability, in production", but closer to -O1, so real optimizations do happen which make debugging harder. 

Also nit: on macOS, it's -glldb for optimize debugging for lldb =) since gdb on macOS is mostly broken.